### PR TITLE
Added 1503 solutions

### PR DESCRIPTION
**Pull request by WaqarTabish2807**
Added different solutions for the question.

### CODE REVIEW

File: solutions/1500-1599/1503-Last-Moment-Before-All-Ants-Fall-Out-of-a-Plan.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+---
+description: 'Author: @wingkwong | https://leetcode.com/problems/last-moment-before-all-ants-fall-out-of-a-plank/'
```
Comment (wingkwong) [inline solutions/1500-1599/1503-Last-Moment-Before-All-Ants-Fall-Out-of-a-Plan.md:2]: Don't use my name

**Review by wingkwong — CHANGES_REQUESTED** [review]
Run the app and verify locally first please. Can't even start the app with your broken changes.

**Comment by WaqarTabish2807** [comment]
I ran these on leetcode locally, and all the codes are running without an error. Let me know what changes does it require?

**Comment by wingkwong** [comment]
Not your code. Run the app locally.

image

**Comment by WaqarTabish2807** [comment]
hey, @wingkwong I am unable to get your comment. Can you please brief it?

**Comment by wingkwong** [comment]
@WaqarTabish2807 I build this site using docusaurus. When you fork my repository, you need to clone it to your local machine. Then navigate to the root and run those commands to start the development server. When you start it, a browser window will open at http://localhost:3000/. I could see the error thrown from your page. 

You should verify the changes there, not in a markdown editor.

```bash
cd leetcode-the-hard-way
npm i 
npm run start
# By default, a browser window will open at http://localhost:3000/.
```

**Pull request closed without merging**